templates/docker-compose.yml.j2

```diff
@@ -56,6 +56,9 @@ services:
 {% endif %}
 {% if "rln-relay" in nim_waku_protocols_enabled %}
       --rln-relay-content-topic={{ nim_waku_rln_relay_content_topic }}
+      --rln-relay-dynamic={{ nim_waku_rln_relay_dynamic }}
+      --eth-account-address={{ nim_waku_eth_account_address }}
+      --eth-mem-contract-address={{ nim_waku_eth_mem_contract_address }}
```

Comment:
If those flags are related to `rln-relay` why are they not prefixed with `--rln-relay-`?

Reply:
Tracking that issue here - https://github.com/status-im/nwaku/issues/1196